packages/tasks/src/local-apps.ts

```diff
@@ -207,10 +207,28 @@ curl -X POST "http://localhost:8000/v1/completions" \\
 		"temperature": 0.5
 	}'`;
 	const runCommand = model.tags.includes("conversational") ? runCommandInstruct : runCommandNonInstruct;
+
+	let setup;
+	let dockerCommand;
+	
+	if (model.tags.includes("mistral-common")) {
+		setup = [
+			"# Install vLLM from pip:",
+			"pip install vllm",
+			"# Make sure you have the latest version of mistral-common installed:",
+			"pip install --upgrade mistral-common"
+		].join("");
+		dockerCommand = `# Load and run the model:\ndocker exec -it my_vllm_container bash -c "vllm serve ${model.id} --tokenizer_mode mistral --config_format mistral --load_format mistral --tool-call-parser mistral --enable-auto-tool-choice"`;
```

Comment:
out of curiosity, what happens if one runs `docker exec -it my_vllm_container bash -c "vllm serve ${model.id}"` instead of having all the mistral arguments? In general we avoid putting too many options in the snippets like this

Reply:
It wouldn't know how to load the model correctly so failing :/

Reply:
wouldn't this be something to fix on vllm side directly? feels weird if all vllm-compatible models can be correctly served with  `vllm server <model id>` except the mistral-common ones that requires a very specific command

(not against adding it, just wondering why it's not a default)